.github/workflows/checkov.yaml

```diff
@@ -1,20 +1,25 @@
-name: "Checkov GitHub Action"
-on:
+# yamllint disable rule:line-length
+# Use template from https://github.com/honestbank/workflows/tree/main/examples/caller-workflows
+---
+name: "Checkov Repository Workflow"
+permissions: read-all
+
+on:  # yamllint disable-line rule:truthy
   pull_request:
-    branches: [test, dev, qa, prod, main]
+    branches:
```

Comment:
You can see this is a component module using trunk based git branching off of the branch `main`, so this needs to include `main`.  I think it's fine to have the same file in all repos, so `test`, `dev`, `qa`, `prod` and `main` should be the triggers for all workflows.

Reply:
Not just this workflow, but the others too.